homeassistant/components/fritzbox/sensor.py

```diff
@@ -87,7 +87,9 @@ class FritzSensorEntityDescription(
         device_class=SensorDeviceClass.POWER,
         state_class=SensorStateClass.MEASUREMENT,
         suitable=lambda device: device.has_powermeter,  # type: ignore[no-any-return]
-        native_value=lambda device: device.power / 1000 if device.power else 0.0,
+        native_value=lambda device: round(device.power / 1000, 3)
+        if device.power
+        else 0.0,
```

Comment:
Try to avoid an inline statement crossing multiple lines. Instead, extract this into a separate method and use that in the lambda.

Reply:
maybe this fits?

`native_value=lambda device: round(device.power or 0.0 / 1000, 3)`

Reply:
nice, this fits now👍 
I will create another PR to make all other sensor definitions pretty, since there are some multiline lambdas left